sdk/storage/azure-storage-blobs/test/perf/inc/azure/storage/blobs/test/download_blob.hpp

```diff
@@ -42,7 +44,10 @@ namespace Azure { namespace Storage { namespace Blobs { namespace Test {
     {
       // Call base to create blob client
       BlobsTest::Setup();
+      
       long size = m_options.GetMandatoryOption<long>("Size");
+      m_downloadBuffer = std::make_unique<std::vector<uint8_t>>(size);
```

Comment:
Change this to local instance
```suggestion
      // Allocate a buffer in heap of length `size`. The buffer is destroyed when this instance is out of scope.
      std::vector<uint8_t> downloadBuffer(size);
```